userland/src/async_parameters.cpp

```diff
@@ -84,6 +84,11 @@ int main(int argc, char ** argv)
     std::cout << "Parameter prefix: " << parameter_prefix << std::endl;
   }
 
+  auto callback = [](const std::shared_ptr<rcl_interfaces::msg::ParameterEvent> & event) -> void {
+      std::cout << "parameter foo updated" << std::endl;
```

Comment:
Fixed. I ended up printing all the information from ParameterEvent (new, changed and deleted parameters)